.github/semantic-release.md

```
@@ -67,3 +67,56 @@ ci(actions): bump actions/checkout from 7 to 8    # nothing
+  release rules match on and would release nothing.
+
+Merge commits, `fixup!` commits and git's own `Revert "..."` subjects are
+ignored, as they are by every conventional-commits parser. The pull request
+title is not checked: pull requests here are merged, not squashed, so the title
+never enters the history. Turning squash merging on would make the title the
+commit subject, and would need a check of its own.
+
+To lint a range by hand, the way the workflow does:
+
+```console
+$ npm install --no-save @commitlint/cli@21 @commitlint/config-conventional@21
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)